rest/route/host_agent.go

```diff
@@ -153,6 +157,17 @@ func (h *hostAgentNextTask) Run(ctx context.Context) gimlet.Responder {
 				"agent":         evergreen.AgentVersion,
 				"current_agent": h.host.AgentRevision,
 			}))
+			event.LogHostScriptExecuteFailed(h.host.Id, err)
```

Comment:
Since this is a pretty specific event case for quarantined hosts, can we use a more specific event log type like `LogHostAgentCleanupFailed` rather than reuse the host script event? There are other host scripts that run which are unrelated to this one.

If we go this route, we also have to file a UI ticket to add it to the host event logs.

Reply:
I've made a change to instead introduce a new counter field on the host that gets reset to zero when its running task gets cleared.